connexion/decorators/produces.py

```diff
@@ -30,6 +32,9 @@ def default(self, o):
         if isinstance(o, datetime.date):
             return o.isoformat()
 
+        if isinstance(o, Decimal):
+            return float(o)
```

Comment:
We can fallback to strings for precise representation, but I am not sure if this is the desired ``default`` behavior. I felt using floats (👿 ) is somehow expected/acceptable, unless precision is required which in this case dev would use their own implementation.